drawBot/drawBotDrawingTools.py

```diff
@@ -1978,6 +1978,33 @@ def imagePixelColor(self, path, xy):
         color = color.colorUsingColorSpaceName_("NSCalibratedRGBColorSpace")
         return color.redComponent(), color.greenComponent(), color.blueComponent(), color.alphaComponent()
 
+    def imageResolution(self, path):
+        """
+        Return the image resolution for a given image.
+        """
+        if isinstance(path, AppKit.NSImage):
+            # its an NSImage
+            # get the bitmap representation
+            try:
+                rep = path.representations()[0]
+            except Exception:
```

Comment:
Isn't it a simple IndexError you want to catch here?

Reply:
fixed